<!-- -->
<launch>
  <arg name="map"  default="playground"/>

  <node name="map_server"
  pkg ="map_server"
  type="map_server"
  args="$(find upc_mrn)/maps/$(arg map).yaml"/>

</launch>